files/ko/web/api/web_audio_api/simple_synth/index.md

```diff
@@ -510,7 +510,7 @@ function playTone(freq) {
 
 oscillator의 주파수는 {{domxref("Oscillator.frequency")}} {{domxref("AudioParam")}} 객체의 값을 설정함으로써 `freq` 파라미터에 명시된 값으로 설정됩니다. 그리고서, 마침내, oscillator는 상속된 {{domxref("AudioScheduledSourceNode.start()")}} 메서드를 호출하여 소리를 생성하도록 시작됩니다.
 
-#### 음 재생하기
+#### 음을 연주하다
```

Comment:
Please check the translation, I'm not sure

Reply:
- https://github.com/mdn/translated-content/pull/17716/commits/37dd697c6bd5ce59afeac93b93b50008640fa27f